tests/unit_test/chat/user_media_test.py

```
@@ -0,0 +1,363 @@
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)